src/p2p/mod.rs

```diff
@@ -17,13 +17,20 @@ mod transport;
 pub use addr::{MultiaddrWithPeerId, MultiaddrWithoutPeerId};
 pub use {behaviour::KadResult, swarm::Connection};
 
+/// An IPFS swarm.
```

Comment:
```suggestion
/// Type alias for [`libp2p::Swarm`] running the [`behaviour::Behaviour`] with the given [`IpfsTypes`].
```

Not sure if "running" is the correct verb here. All in all this type alias might be a bit unnecessary.